tests/build/depfile_package.srctree

```diff
@@ -7,18 +7,21 @@ PYTHON package_test.py
 
 import os.path
 
+def getpath(*args):
```

Comment:
Maybe rename this to `pkgpath()`? `getpath()` seems overly generic.

Reply:
Good point, done.